cluster-provision/gocli/cmd/provision.go

```diff
@@ -334,3 +355,9 @@ func _cmd(cli *client.Client, container string, cmd string, description string)
 	}
 	return nil
 }
+
+func perform_phase(cli *client.Client, container string, script string, envVars string) error {
```

Comment:
At golang is more idiomatic `performPhase`

Reply:
u right i was in python in my mind ;]